SKILL.tsv

```diff
@@ -4410,7 +4410,7 @@ SKILL_20150714_004409	$Increases the damage dealt on an enemy with [Fire Pillar]
 SKILL_20150714_004410	$Increases the damage dealt on an enemy with [Hell Breath] by 1% per attribute level.
 SKILL_20150714_004411	{memo X}$Deals fire damage to enemies hit by [Fireball] for 5 seconds with a chance of 10% per attribute level.
 SKILL_20150714_004412	{memo X}When an ally steps on [Fire Wall], Fire ATK will increase by 5 per Attribute level for 5 secs.
-SKILL_20150714_004413	$Knockbacks enemies hit by [Hell Breath] with a 10% chance per attribute level.
+SKILL_20150714_004413	$Knocked back enemies hit by [Hell Breath] with a 10% chance per attribute level.
```

Comment:
@stvkno  no, [Gust] will definitely knock enemies back as an innate property of the skill. It's describing that the enemies (which are already knocked back) is now more vulnerable to Magic Attacks.